src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2623,14 +2623,20 @@ void InitSolarHeatGains(EnergyPlusData &state)
                                 state.dataSurface->SurfReflFacBmToDiffSolGnd)); // For linear indexing
         Real64 GndReflSolarRad = 0.0;
         Real64 GndSolarRadInc = max(state.dataEnvrn->BeamSolarRad * state.dataEnvrn->SOLCOS(3) + state.dataEnvrn->DifSolarRad, 0.0);
+
+        for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
+            state.dataSurface->Surface(SurfNum).IncSolMultiplier = GetSurfIncidentSolarMultiplier(state, SurfNum);
+        }
         for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
-            state.dataSurface->SurfSkySolarInc(SurfNum) = state.dataEnvrn->DifSolarRad * state.dataSolarShading->SurfAnisoSkyMult(SurfNum);
+            auto const &SurfIncSolarMultiplier = state.dataSurface->Surface(SurfNum).IncSolMultiplier;
```

Comment:
`state.dataSurface->Surface(SurfNum).IncSolMultiplier` is just a float right?  If so, we for sure don't need the `auto` shorthand.  And I don't _think_ we really want it to be a reference either.  I guess neither of these are really functional changes, so I don't think it's worth holding this up.  I'll keep moving.

Reply:
Yes. It is just a real number. Should just use Real64 instead of the ref. I'll change it now.

Reply:
Thanks, there was also something in the Clang-Format test, so get that fixed up too and this could be ready assuming the >1 comment is resolved.